lib/safira_web/controllers/redeem_controller.ex

```diff
@@ -25,23 +28,26 @@ defmodule SafiraWeb.RedeemController do
 
   defp company_aux(conn, redeem_params, user) do
     redeem_params = Map.put(redeem_params, "badge_id", user.company.badge_id)
+
     with {:ok, %Redeem{} = _redeem} <- Contest.create_redeem(redeem_params) do
       conn
       |> put_status(:created)
-      |> json(%{redeem: "Badge redeem successfully"})
+      |> json(%{redeem: "Badge redeem successfully. Tokens added to your balance"})
```

Comment:
Either `Badge redeem succesful` or 'Badge redeemed successfully'. The current form has incorrect grammar.

Reply:
As you can see, line 31 was there before my update. The current and older form has incorrect grammar, yes. But that was not done by me. No problem updating it.